app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,35 @@
+class CommentsController < ApplicationController
+  def index; end
+
+  def show; end
+
+  def new; end
+
+  def create
+    @post = Post.find_by id: params[:id]
+    @comment = @post.comments.create comment_params
+    if @comment.save
+      flash[:success] = t "comments.create"
+    else
+      flash.now[:warning] = t "comments.createfail"
+    end
+    redirect_to post_path @post
+  end
+
+  def destroy
+    @post = Post.find_by id: params[:id]
+    @comment = @post.comments.find params[:id]
```

Comment:
I don't understand if you want to use 'params[:id]' for 'Post' or 'Comment' table ?

Reply:
I not sure how to create comment for post so i am thinking to find it to show